apps/resources/urls.py

```diff
@@ -9,4 +10,9 @@
     url(r'^add/$', ResourceCreateView.as_view(), name='resource-create'),
     url(r'^thank-you/$', PageDetailView.as_view(), name='resource-thank-you'),
     url(r'^(?P<slug>[\w-]+)/$', ResourceDetailView.as_view(), name='resource-detail'),
+    url(
+        r'^(?P<slug>[\w-]+)/(?P<id>[0-9]+)/$',
```

Comment:
Could we give this a slightly better URL? Something which mentions report maybe.